### DISCUSSION TITLE
obs-rtspserver v3.1.0-beta1

### DISCUSSION THREAD

**Discussion by iamscottxu** [iamscottxu/obs-rtspserver#141] [General]
## 🚀 New Features
* **rtsp-server**: Added frame auto-drop feature to solve the problem of delay accumulation.([#102](https://github.com/iamscottxu/obs-rtspserver/issues/102))
* **ui**: Added dropped frames and total frames display.([#102](https://github.com/iamscottxu/obs-rtspserver/issues/102))
* **locale**: Added translation for "Dropped frames".

## 💊 Bugfixes
* **rtsp-output**: Fixed the bug that the SPP header could not be obtained when using the jim-nvenc encoder. ([#128](https://github.com/iamscottxu/obs-rtspserver/issues/128))

## 🏎️ Performance Improvements
* **ui**: Optimized the code.
* **rtsp-server**: Optimized the code.
* **locale**: Improved "Total Data Output" and "Bitrate" translation in zh-TW.

**Full Changelog**: https://github.com/iamscottxu/obs-rtspserver/compare/v3.0.0...v3.1.0-beta1

<hr /><em>This discussion was created from the release <a href='https://github.com/iamscottxu/obs-rtspserver/releases/tag/v3.1.0-beta1'>obs-rtspserver v3.1.0-beta1</a>.</em>

**Comment by goku1982jason** [comment]
I have for the past few days tried to get this to stream either locally on the same computer or on any network computers with both this and the 3.0 version with no luck. Are there any tutorials out there that can point me in the right direction?

**Reply by ivorobeychik-cloudinary** [reply]
Same here , not sure of how to use it , or once installed if there any options in OBS to select ...

**Comment by BraceMarin** [comment]
![Sin título](https://github.com/iamscottxu/obs-rtspserver/assets/91663030/08a62067-a280-480d-89be-189b6b0bbbc4)

"I'm trying to use the plugin; however, in all current versions of it (3.0 and beta 3.1), I'm getting the same error, and it's not working. The specific error message says 'Failed to initialize data acquisition.' I've installed OBS 27 and plugin 2.1.2, and it works perfectly. How can this be resolved?"
